src/styles/blocks/default-styles.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Halvetica, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
```

Comment:
you don't have to reset margins/paddings for everything on the page, preferably to do it only for certain tags

P.S. it's ok to use box-sizing with *